### PR TITLE
Changed the way a few sentences were written

### PR DESCRIPTION
**Pull request by ObliviousParadigm**
Added comma before 'and'

- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Comment by ObliviousParadigm** [comment]
Hey! This is my first PR. The Travis CI failed to build. I don't know what to do now.

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 ## What is it?
 
-**pandas** is a Python package providing fast, flexible, and expressive data
+**pandas** is a Python package that provides fast, flexible, and expressive data
```
Comment (simonjayhawkins) [inline README.md:23]: This sentence is also in LONG_DESCRIPTION of setup.py. should probably keep in sync so can you change there as well?

**Comment by simonjayhawkins** [comment]
> Hey! This is my first PR. The Travis CI failed to build. I don't know what to do now.

this is a known issue, xref #34721

File: README.md (hunk 1 above)
Comment (ObliviousParadigm) [reply README.md:23]: Got it. I made the changes. It's being checked right now.

Comment (ObliviousParadigm) [reply README.md:23]: Hey! I've made the necessary changes.

**Review by simonjayhawkins — APPROVED** [review]
Thanks @ObliviousParadigm lgtm

**Comment by WillAyd** [comment]
Thanks @ObliviousParadigm

**Comment by ObliviousParadigm** [comment]
Thank you! Also, I don't know how else to contribute. Could I please get some pointers on what I can do next?

**Comment by WillAyd** [comment]
A good place to start is probably the good first issues on GitHub:

https://github.com/pandas-dev/pandas/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22

If you see one that interests you (even without that label) typically a good way to start

**Pull request merged**